app/views/static_pages/help.html.erb

```diff
@@ -1,2 +1,3 @@
+<% provide :title, t(".title") %>
 <h1><%= t ".title" %></h1>
 <%= raw t ".content" %>
```

Comment:
em có biết về `raw` không ? phân biệt nó với các method cùng loại khác thử đi em

Reply:
raw là để viết đoạn html chưa đc mã hoá. còn method cùng loại với nó thì mình chưa tìm hiểu là có loại nào. nếu là html_safe. thì html_safe nó bảo toàn thẻ html.